src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Services/CohortDomainServiceTests.cs

```diff
@@ -564,7 +564,7 @@ public CohortDomainServiceTestFixture()
                 Db.ChangeOfPartyRequests.Add(ChangeOfPartyRequest.Object);
 
                 AcademicYearDateProvider = new Mock<IAcademicYearDateProvider>();
-                AcademicYearDateProvider.Setup(x => x.CurrentAcademicYearEndDate).Returns(new DateTime(2020, 7, 31));
+                AcademicYearDateProvider.Setup(x => x.CurrentAcademicYearEndDate).Returns(new DateTime(2022, 7, 31));
```

Comment:
Is it worth doing a `DateTime.Now.AddYears(1)` on this so it's not changed again in 2022?